### ISSUE TITLE
Added it (Italian) translation

### ISSUE THREAD

**Issue by airon90** [vanilla-music/vanilla-music-lyrics-search#8]
I have translated your application into `it` (*Italian*). Here is the xml:
```xml
<resources>
    <string name="app_name">Ricerca testi di Vanilla</string>
    <string name="lyrics_search">Cerca testo</string>
    <string name="plugin_desc">Ti permette di cercare i testi delle canzoni su LyricWiki</string>
    <string name="lyrics_not_found">Testo non trovato!</string>
    <string name="write">Scrivi...</string>
    <string name="write_to_lrc">Scrivi su un file .lrc</string>
    <string name="write_to_tag">Scrivi nel tag del file</string>
    <string name="file_written_successfully">File scritto con successo!</string>
    <string name="error_writing_file">Errore durante la scrittura del file:</string>
    <string name="reload">Ricarica</string>
</resources>

```

---

*Translation made with [Stringlate](https://github.com/LonamiWebs/Stringlate).*

**Comment by Kaned1as** [comment]
Wow that's a bummer... Will pick this once I'm home

**Comment by Kaned1as** [comment]
Merged in b78635c

**Comment by Kaned1as** [comment]
Will be available in 1.2.1
